frontend/src/App.css

```diff
@@ -2,7 +2,7 @@
   margin: 0 auto;
   text-align: center;
   height: 100dvh;
-  width: 100dvw;
```

Comment:
`dvw` doesn't account for the space taken up by the scrollbar in its width calculations. This resulted in the width being ~15px too much and causing a horizontal scrollbar to appear.